recipe/activate-winsdk.sh

```diff
@@ -15,79 +15,84 @@ echo "By setting CONDA_BUILD_WINSDK, you are agreeing to the terms and condition
+        echo "Making symlinks for case-sensitive filesystems"
+        # Make symlinks for libraries
+        for f in $(find ${WINSDK_DIR}/lib/um/x64 -name "*.[L|l]ib"); do
+            name=$(basename $f)
+            full_lower=$(echo "$name" | awk '{print tolower($0)}')
+            if [[ "$name" != "$full_lower" ]]; then
+                ln -sf "$f" "$(dirname $f)/$full_lower"
+            fi
+            lib_lower="${name:0:${#name} - 4}.lib"
+            if [[ "$lib_lower" != "$name" && "$lib_lower" != "$full_lower" ]]; then
+                ln -sf "$f" "$(dirname $f)/$lib_lower"
+            fi
+        done
 
-      # Create a VFS overlay as the Win SDK assumes case insensitive file system
-      echo "version: 0" > winsdk_vfs_overlay.yaml
-      echo "case-sensitive: false" >> winsdk_vfs_overlay.yaml
-      echo "roots:" >> winsdk_vfs_overlay.yaml
+        # Create a VFS overlay as the Win SDK assumes case insensitive file system
+        echo "version: 0" > winsdk_vfs_overlay.yaml
+        echo "case-sensitive: false" >> winsdk_vfs_overlay.yaml
+        echo "roots:" >> winsdk_vfs_overlay.yaml
 
-      for dir in $(find ${WINSDK_DIR}/include -type d); do
-          files=$(find $dir -maxdepth 1 -name "*.h")
-          if [[ "$files" != "" ]]; then
-              echo "  - name: \"$dir\"" >> winsdk_vfs_overlay.yaml
-              echo "    type: directory" >> winsdk_vfs_overlay.yaml
-              echo "    contents:" >> winsdk_vfs_overlay.yaml
-              for f in $files; do
-                  echo "      - name: \"$(basename $f)\"" >> winsdk_vfs_overlay.yaml
-                  echo "        type: file" >> winsdk_vfs_overlay.yaml
-                  echo "        external-contents: \"$f\"" >> winsdk_vfs_overlay.yaml
-              done
-          fi
-      done
+        for dir in $(find ${WINSDK_DIR}/include -type d); do
+            files=$(find $dir -maxdepth 1 -name "*.h")
+            if [[ "$files" != "" ]]; then
+                echo "  - name: \"$dir\"" >> winsdk_vfs_overlay.yaml
+                echo "    type: directory" >> winsdk_vfs_overlay.yaml
+                echo "    contents:" >> winsdk_vfs_overlay.yaml
+                for f in $files; do
+                    echo "      - name: \"$(basename $f)\"" >> winsdk_vfs_overlay.yaml
+                    echo "        type: file" >> winsdk_vfs_overlay.yaml
+                    echo "        external-contents: \"$f\"" >> winsdk_vfs_overlay.yaml
+                done
+            fi
+        done
 
-      # Add symlinks for LLVMs WinMSVC.cmake
-      ln -sf ${WINSDK_DIR}/include ${WINSDK_DIR}/Include
-      ln -sf ${WINSDK_DIR}/lib ${WINSDK_DIR}/Lib
-    fi
-    rm -rf win10sdk_iso
-    rm -rf win10sdk.iso
-  popd
+        # Add symlinks for LLVMs WinMSVC.cmake
+        ln -sf ${WINSDK_DIR}/include ${WINSDK_DIR}/Include
+        ln -sf ${WINSDK_DIR}/lib ${WINSDK_DIR}/Lib
+      fi
+      rm -rf win10sdk_iso
+      rm -rf win10sdk.iso
+      pushd
+    popd
+  )
```

Comment:
Looking at this with whitespace changes ignored, you seem to be adding an extraneous `pushd` here?